static/app/utils/performance/suspectSpans/types.tsx

```diff
@@ -39,3 +39,8 @@ export type SpanOp = {
 };
 
 export type SpanOps = SpanOp[];
+
+export type SpanSlug = {
+  op: string;
+  group: string;
```

Comment:
@Zylphrex Is `group` attribute/field here referring to the span id?

Reply:
Yes